pkg/types/defaults/installconfig.go

```diff
@@ -24,7 +24,7 @@ var (
 	defaultServiceNetwork = ipnet.MustParseCIDR("172.30.0.0/16")
 	defaultClusterNetwork = ipnet.MustParseCIDR("10.128.0.0/14")
 	defaultHostPrefix     = 23
-	defaultNetworkType    = string(operv1.NetworkTypeOpenShiftSDN)
+	defaultNetworkType    = string(operv1.NetworkTypeOVNKubernetes)
```

Comment:
Logic around https://github.com/openshift/installer/blob/master/pkg/types/defaults/installconfig.go#L50-L56 also needs to be cleaned up.

Reply:
It's updated now @sadasu, I think it is much simpler now right, and just set to ovnk if ""